MaxRectsBinPack.cpp

```diff
@@ -25,13 +25,14 @@ binHeight(0)
 {
 }
 
-MaxRectsBinPack::MaxRectsBinPack(int width, int height)
+MaxRectsBinPack::MaxRectsBinPack(int width, int height, bool allowFlip) : m_AllowFlip(allowFlip)
 {
 	Init(width, height);
```

Comment:
This should call `Init(width, height, allowFlip);` instead, since otherwise it overrides allowFlip to true always.